src/components/Alteration/Articles/CompanyProvisions.vue

```diff
@@ -230,35 +231,37 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
   border-right: 1px solid $gray1;
 }
 
-::v-deep .v-input--checkbox .theme--light.v-icon {
-  margin-top: 0;
-  margin-left: 0;
-}
+:deep() {
+  .v-input--checkbox .theme--light.v-icon {
+    margin-top: 0;
+    margin-left: 0;
+  }
 
-::v-deep .invalid.v-input--checkbox .theme--light.v-icon {
-  color: $app-red;
-  margin-top: 0;
-  margin-left: 0;
-}
+  .invalid.v-input--checkbox .theme--light.v-icon {
+    color: $app-red;
+    margin-top: 0;
+    margin-left: 0;
+  }
 
-::v-deep .v-input--checkbox .theme--light.v-label {
-  line-height: 1.375rem;
-  font-size: $px-14;
-  font-weight: normal;
-}
+  .v-input--checkbox .theme--light.v-label {
+    line-height: 1.375rem;
+    font-size: $px-14;
+    font-weight: normal;
+  }
 
-::v-deep .invalid.v-input--checkbox .theme--light.v-label {
-  color: $app-red;
-}
+  .invalid.v-input--checkbox .theme--light.v-label {
+    color: $app-red;
+  }
 
-::v-deep #checkbox-div .v-input__slot {
-  align-items: start;
-  justify-content: start;
-}
+  #checkbox-div .v-input__slot {
+    align-items: start;
+    justify-content: start;
+  }
 
-::v-deep .v-input--selection-controls {
-  margin-top: 1.875rem;
-  padding: 0;
+  .v-input--selection-controls {
+    margin-top: 1.875rem;
+    padding: 0;
+  }
```

Comment:
I combined these all into a single `:deep()`. It works and it looks cleaner to me. Maybe there's even a performance boost since the framework only has to record one deep record instead of 6?